docs/install-upgrade/build-wiring.md

```diff
@@ -107,14 +177,105 @@ Fabric connections serve as connections between switches, they form the fabric o
 
 VPCs need VPC Peerings to talk to each other. VPC Peerings come in two varieties: local and remote.
 
+``` mermaid
+graph TD
+    S1([Spine 1])
+    S2([Spine 2])
+    L1([Leaf 1])
+    L2([Leaf 2])
+    TS1[Server1]
+    TS2[Server2]
+    TS3[Server3]
+    TS4[Server4]
+
+    S1 & S2 <--> L1 & L2
+    L1 <--> TS1 & TS2
+    L2 <--> TS3 & TS4
+
+
+    subgraph VPC 1
+    TS1
+    TS2
+    end
+
+    subgraph VPC 2
+    TS3
+    TS4
+    end
+```
+
 #### Local VPC Peering
 
 When there is no dedicated border/peering switch available in the fabric we can use local VPC peering. This kind of peering tries sends traffic between the two VPC's on the switch where either of the VPC's has workloads attached. Due to limitation in the Sonic network operating system this kind of peering bandwidth is limited to the number of VPC loopbacks you have selected while initializing the fabric. Traffic between the VPCs will use the loopback interface, the bandwidth of this connection will be equal to the bandwidth of port used in the loopback.
 
+``` mermaid
+graph TD
+
+    L1([Leaf 1])
+    S1[Server1]
+    S2[Server2]
+    S3[Server3]
+    S4[Server4]
+
+    L1 <-.2,loopback.-> L1;
+    L1 <-.3.-> S1;
+    L1 <--> S2 & S4;
+    L1 <-.1.-> S3;
```

Comment:
Could this diagram have explanations for arrows 1, 2, 3, like you added below the last diagram on the page?

Reply:
done!